docs/recommended-setup.md

```diff
@@ -0,0 +1,37 @@
+---
+id: recommended-setup
+title: The Recommended Setup
+sidebar_label: Overview
+---
+
+Single-spa itself is not opinionated about your build tools, CI process, or local development workflow. However, to implement single-spa you will have to figure all of those things out (and more). To help you decide how to approach these problems, the single-spa core team has put together a "recommended setup" that gives an opinionated approach to solving the practical problems of microfrontends.
+
```

Comment:
I love the direction this is going. I'd still like to keep the text summary from before as the headline content here with the videos being supplemental